sympy/simplify/tests/test_cse.py

```diff
@@ -67,7 +70,7 @@ def test_cse_single2():
     substs, reduced = cse(e)
     assert substs == [(x0, x + y)]
     assert reduced == [sqrt(x0) + x0**2]
-    assert isinstance(cse(Matrix([[1]]))[1][0], Matrix)
+    assert isinstance(cse(Matrix([[1]]))[1], Matrix)
```

Comment:
This test change is now breaking downstream code. Why was the test changed? It is generally not a good idea to change existing behavior.